src/tikzplotlib/_path.py

```diff
@@ -469,8 +469,8 @@ def mpl_linestyle2pgfplots_linestyle(data, line_style, line=None):
         default_dashOffset, default_dashSeq = mpl.lines._get_dash_pattern(line_style)
 
         # get dash format of line under test
-        dashSeq = line._us_dashSeq
-        dashOffset = line._us_dashOffset
+        dashOffset = line._dash_pattern[0]
+        dashSeq = line._dash_pattern[1]
```

Comment:
As in https://github.com/nschloe/tikzplotlib/pull/558/files, it would be better to keep this backwards-compatible:
```suggestion
        try:
            dashOffset, dashSeq = line._dash_pattern[:2]
        except AttributeError:
            # backwards-compatibility with matplotlib < 3.7.0
            dashSeq = line._us_dashSeq
            dashOffset = line._us_dashOffset
```